support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -345,6 +349,14 @@ class CreateSubscriptionController(
                 ),
                 body = writeable.toEntity(err.message),
               )
+            case ServerError(code) if code == emailAddressAlreadyTakenCode =>
+              Result(
+                header = new ResponseHeader(
+                  status = INTERNAL_SERVER_ERROR,
+                  reasonPhrase = Some(emailAddressAlreadyTakenCode),
+                ),
+                body = writeable.toEntity(emailAddressAlreadyTakenCode),
```

Comment:
This would make it very easy for us to show a more relevant error message to the user on the client if we wanted.